modules/nw-sriov-installing-operator.adoc

```diff
@@ -2,16 +2,6 @@
 //
 // * networking/hardware_networks/installing-sriov-operator.adoc
 
-ifeval::["{product-version}" == "4.2"]
```

Comment:
@jboxman or @mikemckiernan Can either of you confirm that run-level is no longer needed as of 4.5, and since we're only updating 4.6+ now, that this section is okay to remove? Thanks!

Reply:
@zshi-redhat, do we need this content for 4.6+? Thanks!

Reply:
@jboxman No according to the operator manifests.
We removed the label `openshift.io/run-level` since 4.5.